verifier/src/test/groovy/com/okta/jwt/impl/OktaJWTClaimsVerifierTest.groovy

```diff
@@ -180,8 +204,51 @@ class OktaJWTClaimsVerifierTest {
                 iss: "invalid_testIssuer"
         ])
 
-        TestSupport.expect(BadJWTException) {
+        expect(BadJWTException) {
             verifier.verify(claimsSetId, idTokenContext)
         }
     }
+
+    @Test
+    void IdTokenDecodeNoSubjectTest() {
+
+        JWTClaimsSet claimsSet = new JWTClaimsSet(basicValidClaims + [
+                aud: ["my_clientId"],
```

Comment:
Interesting. In our JWT helper we only ask for a singular audience. I wonder if it makes sense to make this a list since it is already support in these libraries.. thoughts?